### PR TITLE
feat: LT-893 - Exporter dashboard redesign

### PR DESCRIPTION
**Pull request by rorysproule1**

### CODE REVIEW

File: assets/css/components/_tiles.scss [hunk 1]
```diff
@@ -17,44 +17,82 @@ $card-shadow: 3px;
...
 	padding-right: 40px;
+	min-height: 200px!important;
 
 	* {
-		@include govuk-font($size: 19, $weight: bold);
-		margin: 0;
-		padding: 0;
-		color: white!important;
+		@include govuk-font($size: 16);
+		color: black!important;
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:32]: `$govuk-text-colour` not black

File: assets/css/components/_tiles.scss [hunk 2]
```diff
@@ -17,44 +17,82 @@ $card-shadow: 3px;
...
 	p {
-		font-weight: normal;
 		line-height: 1.5;
-		color: lighten($card-color, 50%)!important;
 	}
 
-	&:hover {
-		background-color: darken($card-color, 3.5%);
-		box-shadow: 0 $card-shadow 0 darken($card-color, 30%);
+	a {
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:46]: Remove a, use `govuk-link govuk-link--no-visited-state` instead

File: assets/css/components/_tiles.scss [hunk 3]
```diff
@@ -17,44 +17,82 @@ $card-shadow: 3px;
...
+		color: $govuk-brand-colour!important;
 	}
-
-	&:active {
-		transform: translateY($card-shadow);
+	span {
+		@include govuk-font($size: 14, $weight: bold);
+		margin-left: 5px;
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:51]: `govuk-spacing(1)` not 5

File: assets/css/components/_tiles.scss [hunk 4]
```diff
@@ -17,44 +17,82 @@ $card-shadow: 3px;
...
+		color: white!important;
 	}
 
 	&:focus {
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:55]: Remove

File: assets/css/components/_tiles.scss [hunk 5]
```diff
@@ -17,44 +17,82 @@ $card-shadow: 3px;
...
 		box-shadow: none!important;
 	}
 }
+
+.navigation {
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:60]: Should follow BEM. Should be like `.app-navigation`

File: assets/css/components/_tiles.scss [hunk 6]
```diff
@@ -17,44 +17,82 @@ $card-shadow: 3px;
...
+    position: relative;
+    border-bottom: 1px solid #bfc1c3;
+    width: 100%;
+    float: left;
+	color: #0b0c0c;
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:65]: govuk-colour("light-grey") not #0b0c0c.

Also floats are bad

File: assets/css/components/_tiles.scss [hunk 7]
```diff
@@ -17,44 +17,82 @@ $card-shadow: 3px;
...
+}
+
+.home-link {
+    color: #0b0c0c !important;
+    float: left;
+    display: inline-block;
+    padding: 10px 0px 6px 0px;
+    text-decoration: none;
+	border-bottom: 4px solid transparent;
+	&:hover {
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:75]: Indentation too! We use tabs

File: assets/css/components/_tiles.scss [hunk 8]
```diff
@@ -17,44 +17,82 @@ $card-shadow: 3px;
...
+		border-bottom: 4px solid #bfc1c3;
+	}
+}
+
+.no-bottom-margin {
+	margin-bottom: 25px!important;
+}
+
+.float-left {
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:96]: Remove

File: assets/css/components/_tiles.scss (hunk 6 above)
Comment (jfaracik) [reply assets/css/components/_tiles.scss:65]: image

File: assets/css/components/_tiles.scss [hunk 9]
```diff
@@ -17,44 +17,82 @@ $card-shadow: 3px;
...
+	display: inline-block;
+	padding: 10px 0px 6px 0px;
+	text-decoration: none;
+	border-bottom: 4px solid transparent;
+	&:hover {
+		border-bottom: 4px solid #bfc1c3;
+	}
+}
+
+.no-bottom-margin {
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:92]: Remove! `govuk-!-margin-bottom-0` if you need this

File: assets/css/components/_tiles.scss [hunk 10]
```diff
@@ -17,44 +17,82 @@ $card-shadow: 3px;
...
+	&:hover {
+		border-bottom: 4px solid #bfc1c3;
+	}
+}
+
+.nav-item {
+    color: #0b0c0c;
+	float: right;
+	display: inline-block;
+	padding: 10px 0px 6px 0px;
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:84]: Using govuk-spacing(n) instead of hard setting numbers

File: templates/core/hub.html [hunk 11]
```diff
@@ -1,35 +1,139 @@
...
+	<div class="navigation govuk-body-s">
+		<a href="{% url 'core:home' %}" class="home-link">{% lcs 'hub.Navigation.ACCOUNT_HOME' %}</a>
+		<a href="{% url 'organisation:members:members' %}" class="nav-item" id="profile-link">{% lcs 'hub.Navigation.YOUR_ACCOUNT' %}</a>
+	</div>
+{% endblock %}
+
 {% block body %}
-	<h1 class="govuk-heading-xl govuk-!-margin-bottom-4">
+
+	<h1 class="govuk-caption-xl">
```
Comment (jfaracik) [inline templates/core/hub.html:20]: Don't use two H1s, 

```
<span class="govuk-caption-xl">govuk-caption-xl</span>
<h1 class="govuk-heading-xl">govuk-heading-xl</h1>
```

When in doubt look at the GOV UK Design System https://design-system.service.gov.uk/styles/typography/

File: templates/organisation/base.html [hunk 12]
```diff
@@ -1,7 +1,7 @@
 {% extends 'layouts/base.html' %}
 
 {% block back_link %}
-	{% include 'includes/breadcrumbs.html' with title='core.HubPage.ORGANISATION' %}
+{#	{% include 'includes/breadcrumbs.html' with title='core.HubPage.ORGANISATION' %}#}
```
Comment (jfaracik) [inline templates/organisation/base.html:4]: commented out :)

File: templates/organisation/base.html [hunk 13]
```diff
@@ -13,7 +13,10 @@
 {% endblock %}
 
 {% block body %}
-	<h1 class="govuk-heading-l">{{ organisation.name }}</h1>
+	<h1 class="govuk-caption-xl">
+		{{ organisation.name }}
+	</h1>
+	<h1 class="govuk-heading-l">{{ user.first_name }} {{ user.last_name }}</h1>
```
Comment (jfaracik) [inline templates/organisation/base.html:19]: Two h1s

File: ui_automation_tests/pages/exporter_hub_page.py [hunk 14]
```diff
@@ -2,31 +2,35 @@
 
 
 class ExporterHubPage(BasePage):
-    APPLY_FOR_A_LICENCE_BTN = "a[href*='/apply-for-a-licence/']"
-    APPLICATIONS_BTN = "a[href*='/applications/']"
-    MY_GOODS_BTN = "a[href*='/goods/']"
-    TILE_MANAGE_MY_ORGANISATION_SELECTOR = "a[href='/organisation/']"
-    END_USER_ADVISORY_TILE = "a[href*='/end-users/']"
-    RAISE_HMRC_QUERY_BTN = "a[href*='/raise-a-query/"
+    APPLY_FOR_A_LICENCE_BTN_ID = "apply-link"
```
Comment (jfaracik) [inline ui_automation_tests/pages/exporter_hub_page.py:5]: `BUTTON_APPLY_FOR_A_LICENCE_ID`

File: ui_automation_tests/pages/hub_page.py [hunk 15]
```diff
@@ -6,7 +6,7 @@ class Hub(BasePage):
     SWITCH_LINK = "switch-link"  # ID
     SITES_BTN = "[href*='/sites/']"  # CSS
     APPLICATION_BTN = "a[href*='/applications/']"  # CSS
-    APPLICATIONS_TILE = '.app-tiles [href="/applications/"] p'  # ID
+    APPLICATIONS_TILE = "applications-notifications"  # ID
```
Comment (jfaracik) [inline ui_automation_tests/pages/hub_page.py:9]: If you suffix it with _ID you won't need the comment

File: assets/css/components/_tiles.scss (hunk 1 above)
Comment (rorysproule1) [reply assets/css/components/_tiles.scss:32]: Amended

File: assets/css/components/_tiles.scss (hunk 3 above)
Comment (rorysproule1) [reply assets/css/components/_tiles.scss:51]: Amended

File: assets/css/components/_tiles.scss (hunk 5 above)
Comment (rorysproule1) [reply assets/css/components/_tiles.scss:60]: Amended

File: templates/organisation/base.html (hunk 12 above)
Comment (rorysproule1) [reply templates/organisation/base.html:4]: Removed

File: assets/css/components/_tiles.scss (hunk 9 above)
Comment (rorysproule1) [reply assets/css/components/_tiles.scss:92]: Removed and added

File: assets/css/components/_tiles.scss (hunk 2 above)
Comment (rorysproule1) [reply assets/css/components/_tiles.scss:46]: Added

File: assets/css/components/_tiles.scss (hunk 8 above)
Comment (rorysproule1) [reply assets/css/components/_tiles.scss:96]: Removed

File: templates/core/hub.html (hunk 11 above)
Comment (rorysproule1) [reply templates/core/hub.html:20]: Changed to span

File: assets/css/components/_tiles.scss (hunk 10 above)
Comment (rorysproule1) [reply assets/css/components/_tiles.scss:84]: Using govuk spacing!

File: templates/organisation/base.html (hunk 13 above)
Comment (rorysproule1) [reply templates/organisation/base.html:19]: Fixed

File: assets/css/components/_tiles.scss (hunk 6 above)
Comment (rorysproule1) [reply assets/css/components/_tiles.scss:65]: using light grey and no longer using float

File: ui_automation_tests/pages/hub_page.py (hunk 15 above)
Comment (rorysproule1) [reply ui_automation_tests/pages/hub_page.py:9]: Added

File: ui_automation_tests/pages/exporter_hub_page.py (hunk 14 above)
Comment (rorysproule1) [reply ui_automation_tests/pages/exporter_hub_page.py:5]: Added

File: ui_automation_tests/step_defs/test_notifications.py [hunk 16]
```diff
@@ -11,14 +11,13 @@
...
     api_test_client.cases.add_case_note(context, context.case_id)
-    context.number_of_notifications = Hub(driver).return_number_of_notifications()
 
 
 @then("I can see a notification in application tile")
 def notification_exists(driver, context):
     # Creating an application creates an ecju-query attached to it,
     # and we add a case_note, should expect 2 new notifications
-    assert "You have" in Hub(driver).get_text_of_application_tile()
+    assert "2" in Hub(driver).get_text_of_application_tile()
```
Comment (Mark-Spiers-kainos) [inline ui_automation_tests/step_defs/test_notifications.py:20]: Would this number not change over time?

File: applications/services.py [hunk 17]
```diff
@@ -44,6 +44,15 @@ def get_applications(request, page: int = 1, submitted: bool = True):
     return data.json()
 
 
+def get_existing_applications_and_licences(request):
```
Comment (jfaracik) [inline applications/services.py:47]: Could this be `has_existing_applications_and_licences` and just have it return a bool?

File: assets/css/components/_tiles.scss [hunk 18]
```diff
@@ -17,44 +17,76 @@ $card-shadow: 3px;
...
+	background-color: govuk-colour("light-grey");
 	padding: govuk-spacing(5) govuk-spacing(5) govuk-spacing(3);
+	border-top: 4px solid $govuk-brand-colour;
 	box-sizing: border-box;
-	text-decoration: none;
-	box-shadow: 0 $card-shadow 0 darken($card-color, 20%);
 	@include govuk-media-query($from: tablet) {
 		min-height: 150px;
 	}
 	padding-right: 40px;
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:27]: govuk-spacing(8) please :)

Comment (rorysproule1) [reply assets/css/components/_tiles.scss:27]: Fixed

File: applications/services.py (hunk 17 above)
Comment (rorysproule1) [reply applications/services.py:47]: changed to `has_existing_applications_and_licences` and is returning bools

File: assets/css/components/_tiles.scss [hunk 19]
```diff
@@ -17,44 +17,76 @@ $card-shadow: 3px;
...
+	border-bottom: 4px solid transparent;
+	&:hover {
+		border-bottom: 4px solid govuk-colour("mid-grey");
 	}
+}
 
-	&:focus {
-		box-shadow: none!important;
-	}
+notif {
```
Comment (jfaracik) [inline assets/css/components/_tiles.scss:90]: What's notif?

Comment (rorysproule1) [reply assets/css/components/_tiles.scss]: Removedd :)

**Pull request merged**